codelists/open/license.csv

```diff
@@ -1,7 +1,26 @@
-Code,Title,Source
-CC0-1.0,Creative Commons CCZero,https://creativecommons.org/publicdomain/zero/1.0/
-PDDL-1.0,Open Data Commons Public Domain Dedication and License,http://opendatacommons.org/licenses/pddl/1.0/
-CC-BY-4.0,Creative Commons Attribution 4.0,https://creativecommons.org/licenses/by/4.0/
-ODC-By-1.0,Open Data Commons Attribution License,https://opendatacommons.org/licenses/by/1-0/
-CC-BY-SA-4.0,Creative Commons Attribution Share-Alike 4.0,https://creativecommons.org/licenses/by-sa/4.0/
-ODbL-1.0,Open Data Commons Open Database License,https://opendatacommons.org/licenses/odbl/summary/
+Code,Title,Source,OpenDefinitionCategory,Domain,Comments
+CC0-1.0,Creative Commons CCZero,https://creativecommons.org/publicdomain/zero/1.0/,"Content,data",Recommended conformant licenses,Dedicate to the Public Domain (all rights waived) 
+PDDL-1.0,Open Data Commons Public Domain Dedication and License,http://opendatacommons.org/licenses/pddl/1.0/,"Data",Recommended conformant licenses,Dedicate to the Public Domain (all rights waived) 
+CC-BY-4.0,Creative Commons Attribution 4.0,https://creativecommons.org/licenses/by/4.0/,"Content,data",Recommended conformant licenses,
+ODC-By-1.0,Open Data Commons Attribution License,https://opendatacommons.org/licenses/by/1-0/,"Data",Recommended conformant licenses,Attribution for data(bases) 
+CC-BY-SA-4.0,Creative Commons Attribution Share-Alike 4.0,https://creativecommons.org/licenses/by-sa/4.0/,"Content,data",Recommended conformant licenses
+ODbL-1.0,Open Data Commons Open Database License,https://opendatacommons.org/licenses/odbl/summary/,"Data",Recommended conformant licenses,Attribution-ShareAlike for data(bases) 
+CC-BY-{version},Creative Commons Attribution 1.0-3.0,https://opendefinition.org/licenses/cc-by,Other conformant licenses,Content,"Includes all jurisdiction "ports"; Superseded by CC-BY-4.0."
```

Comment:
The problem with having this is that a validation tool will be looking for "CC-BY-{version]" rather than e.g. "CC-BY-3.0". If you want to allow superceded licenses you'll need to list them all individually.